extension/content.js

```diff
@@ -0,0 +1,91 @@
+// Load jQuery and run myCode() after loaded
+;(function () {
+  function loadScript(url, callback) {
+    var script = document.createElement("script")
+    script.type = "text/javascript"
+    if (script.readyState) { //IE
+      script.onreadystatechange = function () {
+        if (script.readyState == "loaded" || script.readyState == "complete") {
+          script.onreadystatechange = null
+          callback()
+        }
+      }
+    } else { //Others
+      script.onload = function () {
+        callback()
+      }
+    }
+    script.src = url
+    document.getElementsByTagName("head")[0].appendChild(script)
+  }
+
+  loadScript("https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js", function () {
+    chrome.extension.sendMessage({request: 'feedMe'}, (result) => {
+      let optionValue = result.option
+      myCode(optionValue)
+    })
+  })
+})();
+
+function myCode(optionValue) {
```

Comment:
OptionValue is passed down from popup.js.  It's value is set by the select option menu in the popup.